api-server/api-server-common/src/storage/impls/postgres/queries.rs

```diff
@@ -929,13 +929,13 @@ impl<'a, 'b> QueryFromConnection<'a, 'b> {
         self.tx
             .query(
                 r#"
-                SELECT pool_id, data
+                SELECT sub.pool_id, data
                 FROM (
-                    SELECT pool_id, data, block_height, ROW_NUMBER() OVER(PARTITION BY pool_id ORDER BY block_height) as oldest
+                    SELECT pool_id, data, pledge_amount, block_height, ROW_NUMBER() OVER(PARTITION BY pool_id ORDER BY block_height DESC) as newest
```

Comment:
this would probably fail anyway but `pledge_amount` column was also changed to `staker_balance`

Reply:
fixed